widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -184,7 +184,11 @@ public synchronized static EngagementLauncher getEngagementLauncher(@Nullable Li
      */
     @NonNull
     public synchronized static EntryWidget getEntryWidget(@Nullable List<String> queueIds) {
-        setupQueueIds(queueIds);
+        if (Dependencies.glia().isInitialized()) {
```

Comment:
What if integrator keeps an `EntryWidget` instance and reuses after initialization?
We will have default queues, instead of provided queues, I think it will be correct to somehow save the queueIds.

Reply:
GitHub shows the diff a little unclear, I will explain what has been changed:
1. Setup queues no matter if SDK is initialized
2. Change log level for this case to 'error'
3. Moved the setupQueueIds function closer to the code which calls it.